unic/ucd/bidi/src/bidi_class.rs

```diff
@@ -215,6 +214,8 @@ impl TotalCharProperty for BidiClass {
 }
 
 
+// UCD/extracted/DerivedBidiClass.txt:
+// "All code points not explicitly listed for Bidi_Class have the value Left_To_Right (L)."
```

Comment:
nit: Feel free to markdown up this and make doc block.

Reply:
👌